lib/capybara/helpers.rb

```diff
@@ -81,6 +81,16 @@ def filter_backtrace(trace)
       new_trace.first.split(/:in /, 2).first
     end
 
+    # Workaround for emulating `warn '...', uplevel: n` in Ruby 2.5 or lower.
+    def warn(message, uplevel: 0)
```

Comment:
Everywhere this is called passes 1 - so just default to 1 and don't pass it

Reply:
Sure! I updated this PR.